chpl/chpl-service/src/main/java/gov/healthit/chpl/app/SurveillanceOversightReportWeeklyApp.java

```diff
@@ -1,115 +1,99 @@
-		 app.setCpdManager((CertifiedProductDetailsManager)context.getBean("certifiedProductDetailsManager"));
-		 app.setCertifiedProductDAO((CertifiedProductDAO)context.getBean("certifiedProductDAO"));
-		 app.setPresenter((SurveillanceOversightAllBrokenRulesCsvPresenter)context.getBean("surveillanceOversightAllBrokenRulesCsvPresenter"));
-		 app.getPresenter().setProps(props);
-		 app.setMailUtils((SendMailUtil)context.getBean("SendMailUtil"));
-		 
-		 //where to store this file
-        String downloadFolderPath;
-        if (args.length > 0) {
-        	downloadFolderPath = args[0];
-        } else {
-        	downloadFolderPath = props.getProperty("downloadFolderPath");
-        }
-        File downloadFolder = new File(downloadFolderPath);
-        if(!downloadFolder.exists()) {
-        	downloadFolder.mkdirs();
-        }
         
-        List<CertifiedProductDetailsDTO> allCertifiedProducts = app.getCertifiedProductDAO().findWithSurveillance();
-        List<CertifiedProductSearchDetails> allCertifiedProductDetails = new ArrayList<CertifiedProductSearchDetails>(allCertifiedProducts.size());
-		for(CertifiedProductDetailsDTO currProduct : allCertifiedProducts) {
-			try {
-				CertifiedProductSearchDetails product = app.getCpdManager().getCertifiedProductDetails(currProduct.getId());
-				allCertifiedProductDetails.add(product);
-			} catch(EntityRetrievalException ex) {
-				logger.error("Could not certified product details for certified product " + currProduct.getId());
-			}
-		}
-		CertifiedProductDownloadResponse allCps = new CertifiedProductDownloadResponse();
-		allCps.setProducts(allCertifiedProductDetails);
+        context.close();
+	}
+	
+	protected void initiateSpringBeans(AbstractApplicationContext context, Properties props){
+		super.initiateSpringBeans(context, props);
+		this.setPresenter((SurveillanceOversightAllBrokenRulesCsvPresenter)context.getBean("surveillanceOversightAllBrokenRulesCsvPresenter"));
+		this.getPresenter().setProps(props);
+	}
+	
+	private void sendOncWeeklyEmail(Set<String> oncWeeklyEmails, Properties props, File downloadFolder, CertifiedProductDownloadResponse cpList) throws IOException, AddressException, MessagingException {
+		String surveillanceReportFilename = null;
+        String htmlMessage = null;
+        String subject = null;
+        File surveillanceReportFile = null;
 		
-        //write out a csv file containing all surveillance
-        String surveillanceReportFilename = downloadFolder.getAbsolutePath() + File.separator + FILENAME;
-        File surveillanceReportFile = new File(surveillanceReportFilename);
+		surveillanceReportFilename = props.getProperty("oversightEmailWeeklyFileName");
+    	surveillanceReportFile = new File(downloadFolder.getAbsolutePath() + File.separator + surveillanceReportFilename);
         if(!surveillanceReportFile.exists()) {
         	surveillanceReportFile.createNewFile();
         } else {
         	surveillanceReportFile.delete();
```

Comment:
It looks like the application functions just fine with lines 86-89 removed. I tested these two scenarios successfully with lines 86-89 commented out (for both send oncDaily/WeeklyEmail() and sendAcbDaily/WeeklySubscriptionEmails():

Test Case 1:
File in \environmentDownloadFiles already exists (has old/different data). Verify that old file is not attached to email.

Test Case 2:
No file exists. Verify that application does not break; new file should be created only once and attached to email.